File: pythonProject/sensor.py
class sensor():
	def __init__(self,Id,ZonaId,VariableMedida,Estado=True):		
		self.Id = Id
		self.ZonaId = ZonaId
		self.VariableMedida = VariableMedida
		self.Estado = Estado		
	#obtener atributos
	def getId(self):
		return self.Id		
	def getZonaId(self):
		return self.ZonaId		
	def getVariableMedida(self):
		return self.VariableMedida		
	def getEstado(self):
		return self.Estado				
	#establecer atributos	
	def setId(self,NewId):
		self.Id = NewId
	def setZonaId(self,NewZonaId):
		self.ZonaId = NewZonaId
	def setVariableMedida(self,NewVariableMedida):
		self.VariableMedida = NewVariableMedida
	def setEstado(self):
		self.Estado = ~self.Estado
		

File: cppProject/sensor.cpp
#include "sensor.h"

//constructor & destructor
sensor::sensor():instrumentacion(){VariableMedida = "null";}

sensor::sensor(const std::string& NewId,const std::string& NewZonaId,bool NewEstado,
			   const std::string& NewVariableMedida):instrumentacion(NewId,NewZonaId,NewEstado){
	VariableMedida = NewVariableMedida;
}

//obtener atributos
inline std::string sensor::getVariableMedida()const{return VariableMedida;}

//establecer atributos
inline void sensor::setVariableMedida(const std::string& NewVariableMedida){ VariableMedida = NewVariableMedida;}